symphony/lib/toolkit/class.mysql.php

```diff
@@ -882,8 +883,9 @@ public function fetchVar($column, $offset = 0, $query = null)
      */
     public function tableContainsField($table, $field)
     {
+        $table = MySQL::cleanValue($table);
+        $field = MySQL::cleanValue($field);
```

Comment:
Ideally, this should be a separate PR, because it has nothing to do with the topic.